mainnet/balance-config-mainnet.json

```diff
@@ -1,446 +1,1169 @@
+        "minBalance": "0.0005",
+        "decimals": "18",
+        "type": "native",
+        "erc20Address": "",
+        "msgPath": "../nativeLowBalanceTemplate.ejs"
+      },
+      {
+        "address": "0xa67bF22Df59071E57b0E345161bCDFcd436eD769",
+        "topic": "MAINNET-relayer-low-balance",
+        "minBalance": "0.0005",
+        "decimals": "18",
+        "type": "native",
+        "erc20Address": "",
+        "msgPath": "../nativeLowBalanceTemplate.ejs"
+      },
+      {
+        "address": "0x8f18154c57D2AFB1E12224788b366c5Da17Ef3E7",
+        "topic": "MAINNET-relayer-low-balance",
+        "minBalance": "0.0005",
+        "decimals": "18",
+        "type": "native",
+        "erc20Address": "",
+        "msgPath": "../nativeLowBalanceTemplate.ejs"
+      },
+      {
+        "address": "0x31404a0ece67Dd8b374C3088a76A5DcAC7D4BF37",
+        "topic": "MAINNET-relayer-low-balance",
+        "minBalance": "0.0005",
+        "decimals": "18",
+        "type": "native",
+        "erc20Address": "",
+        "msgPath": "../nativeLowBalanceTemplate.ejs"
+      },
+      {
+        "address": "0x34D92E33c9973277a83B105B1C45c9579F71C98E",
+        "topic": "MAINNET-relayer-low-balance",
+        "minBalance": "0.0005",
+        "decimals": "18",
+        "type": "native",
+        "erc20Address": "",
+        "msgPath": "../nativeLowBalanceTemplate.ejs"
+      },
+      {
+        "address": "0x580549E46b62929182aCfbCB93544A0f213405a2",
+        "topic": "MAINNET-relayer-low-balance",
+        "minBalance": "0.0005",
+        "decimals": "18",
+        "type": "native",
+        "erc20Address": "",
+        "msgPath": "../nativeLowBalanceTemplate.ejs"
+      },
+      {
+        "address": "0x96c3b060D85F75928CE057AD423BA8EfE778C4d1",
+        "topic": "MAINNET-relayer-low-balance",
+        "minBalance": "0.0005",
+        "decimals": "18",
+        "type": "native",
+        "erc20Address": "",
+        "msgPath": "../nativeLowBalanceTemplate.ejs"
+      },
+      {
+        "address": "0x8CfB285eE7E35a37708D81D019e896e38DA37Bfe",
+        "topic": "MAINNET-relayer-low-balance",
+        "minBalance": "0.0005",
+        "decimals": "18",
+        "type": "native",
+        "erc20Address": "",
+        "msgPath": "../nativeLowBalanceTemplate.ejs"
+      },
+      {
+        "address": "0xE3598C0114dbDD9b953F171b341eA11dd9F72185",
+        "topic": "MAINNET-relayer-low-balance",
+        "minBalance": "0.0005",
+        "decimals": "18",
+        "type": "native",
+        "erc20Address": "",
+        "msgPath": "../nativeLowBalanceTemplate.ejs"
+      },
+      {
+        "address": "0xBEE72DB9034965c02E9F01DBB561050F8713a6F2",
+        "topic": "MAINNET-relayer-low-balance",
+        "minBalance": "0.0005",
+        "decimals": "18",
+        "type": "native",
+        "erc20Address": "",
+        "msgPath": "../nativeLowBalanceTemplate.ejs"
+      },
+      {
+        "address": "0xCcCCA86919b0769D46705e7Adf0EF02e98BebAc7",
```

Comment:
I think this is extra (this is address from Ethereum and there is only on native handler on chain)